File: Desktop/projweb/src/Entity/Regime.php
<?php

namespace App\Entity;

use Doctrine\ORM\Mapping as ORM;

/**
 * Regime
 *
 * @ORM\Table(name="regime")
 * @ORM\Entity
 */
class Regime
{
    /**
     * @var int
     *
     * @ORM\Column(name="IdR", type="integer", nullable=false)
     * @ORM\Id
     * @ORM\GeneratedValue(strategy="IDENTITY")
     */
    private $idr;

    /**
     * @var string
     *
     * @ORM\Column(name="Categorie", type="string", length=255, nullable=false)
     */
    private $categorie;

    /**
     * @var string
     *
     * @ORM\Column(name="Objectif", type="string", length=255, nullable=false)
     */
    private $objectif;

    /**
     * @var string
     *
     * @ORM\Column(name="Image", type="string", length=255, nullable=false)
     */
    private $image;

    public function getIdr(): ?int
    {
        return $this->idr;
    }

    public function getCategorie(): ?string
    {
        return $this->categorie;
    }

    public function setCategorie(string $categorie): self
    {
        $this->categorie = $categorie;

        return $this;
    }

    public function getObjectif(): ?string
    {
        return $this->objectif;
    }

    public function setObjectif(string $objectif): self
    {
        $this->objectif = $objectif;

        return $this;
    }

    public function getImage(): ?string
    {
        return $this->image;
    }

    public function setImage(string $image): self
    {
        $this->image = $image;

        return $this;
    }


}

File: Desktop/projetpidev/connexion3a15/src/connexion3a15/entities/Regime.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package connexion3a15.entities;

/**
 *
 * @author User
 */
public class Regime {
    private int idR;
    
     private String categorie ;
    private  String objectif ;
    private  String image ;
    

    public Regime(int idR, String categorie) {
        this.idR = idR;
        this.categorie = categorie;
    }



    public Regime() {
    }

    public Regime(int idR, String categorie, String objectif, String image) {
        this.idR = idR;
        this.categorie = categorie;
        this.objectif = objectif;
        this.image = image;
    }

    public int getIdR() {
        return idR;
    }

    public void setIdR(int idR) {
        this.idR = idR;
    }

    public String getCategorie() {
        return categorie;
    }

    public void setCategorie(String categorie) {
        this.categorie = categorie;
    }

    public String getObjectif() {
        return objectif;
    }

    public void setObjectif(String objectif) {
        this.objectif = objectif;
    }

    public String getImage() {
        return image;
    }

    public void setImage(String image) {
        this.image = image;
    }

    public Regime(String categorie, String objectif, String image) {
        this.categorie = categorie;
        this.objectif = objectif;
        this.image = image;
    }

    @Override
    public String toString() {
        return "Categorie = "+categorie+" , Objectif ="+objectif;
    }

    
    

    
   
    
}
